src/common/components/StatusCard.jsx

```diff
@@ -126,6 +126,11 @@ const StatusCard = ({ deviceId, position, onClose, disableActions, desktopPaddin
   const positionAttributes = usePositionAttributes(t);
   const positionItems = useAttributePreference('positionItems', 'fixTime,address,speed,totalDistance');
 
+  const serverNavigationAppLink = useSelector((state) => state.session.server.attributes.navigationAppLink);
+  const serverNavigationAppTitle = useSelector((state) => state.session.server.attributes.navigationAppTitle);
+  const userNavigationAppLink = user.attributes.navigationAppLink;
+  const userNavigationAppTitle = user.attributes.navigationAppTitle;
```

Comment:
You should use `useAttributePreference` instead of doing this.

Reply:
Using `useAttributePreference`, if there is a server attribute set, we cannot display the one set in user attributes.

![image](https://github.com/user-attachments/assets/ac7b0c55-cee2-4cf2-b15f-d43d244adc29)

Reply:
I think we should let users decide whether they want to add the navigation link attributes for the whole server, only for some users or both, any opinion?

Reply:
Ok cool, I just tested this so if there are attributes defined on the server side, they will be overridden if the user defines them on their side 👍

Reply:
Fixed ✅